Classes/YLTableViewDataSource.m

```diff
@@ -31,6 +50,22 @@ - (void)reloadVisibleCellForModel:(id)model inTableView:(UITableView *)tableView
   }
 }
 
+- (CGFloat)estimatedHeightForRow:(NSIndexPath *)row inTableView:(UITableView *)tableView {
+  /*!
+   If subclasses want to provide an estimated row height to be used by tableView:estimatedHeightForRowAtIndexPath,
+   they can override this method.
+   For more information, see tableView:estimatedHeightForRowAtIndexPath.
+   */
+  return -1;
```

Comment:
Let's just have this return UITableViewAutomaticDimension and get rid of the branching done in the caller